+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Dec  6 07:01:02 2022 - INFO  - Start environmental preparation.
+ sudo systemctl start firewalld
A dependency job for firewalld.service failed. See 'journalctl -xe' for details.
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Dec  6 07:01:03 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Tue Dec  6 07:01:04 2022 - INFO  - Start executing testcase.
+ sudo firewall-cmd --zone=external --add-masquerade
Error: DBUS_ERROR: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
+ CHECK_RESULT 36
+ actual_result=36
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 36 ']'
+ '[' 0 -eq 0 ']'
+ test 36x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_ip_camouflage.sh line 31'
+ message='oe_test_firewalld_ip_camouflage.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_ip_camouflage.sh line 31'
Tue Dec  6 07:01:08 2022 - ERROR - oe_test_firewalld_ip_camouflage.sh line 31
+ return 0
+ grep yes
+ sudo firewall-cmd --zone=external --query-masquerade
Error: DBUS_ERROR: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_ip_camouflage.sh line 33'
+ message='oe_test_firewalld_ip_camouflage.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_ip_camouflage.sh line 33'
Tue Dec  6 07:01:13 2022 - ERROR - oe_test_firewalld_ip_camouflage.sh line 33
+ return 0
+ sudo firewall-cmd --zone=external --remove-masquerade
Error: DBUS_ERROR: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
+ CHECK_RESULT 36
+ actual_result=36
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 36 ']'
+ '[' 0 -eq 0 ']'
+ test 36x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_ip_camouflage.sh line 35'
+ message='oe_test_firewalld_ip_camouflage.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_ip_camouflage.sh line 35'
Tue Dec  6 07:01:18 2022 - ERROR - oe_test_firewalld_ip_camouflage.sh line 35
+ return 0
+ sudo firewall-cmd --zone=external --query-masquerade
+ grep no
Error: DBUS_ERROR: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_ip_camouflage.sh line 37'
+ message='oe_test_firewalld_ip_camouflage.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_ip_camouflage.sh line 37'
Tue Dec  6 07:01:22 2022 - ERROR - oe_test_firewalld_ip_camouflage.sh line 37
+ return 0
+ LOG_INFO 'Finish testcase execution.'
+ message='Finish testcase execution.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish testcase execution.'
Tue Dec  6 07:01:22 2022 - INFO  - Finish testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Dec  6 07:01:23 2022 - ERROR - failed to execute the case.
+ exit 4
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_firewalld_ip_camouflage.sh post_test'
+ message='oe_test_firewalld_ip_camouflage.sh post_test'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'oe_test_firewalld_ip_camouflage.sh post_test'
Tue Dec  6 07:01:24 2022 - INFO  - oe_test_firewalld_ip_camouflage.sh post_test
